ee/modules/160-multitenancy-manager/crds/projecttemplate.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    heritage: deckhouse
+    module: deckhouse
+  name: projecttemplates.deckhouse.io
+spec:
+  group: deckhouse.io
+  names:
+    kind: ProjectTemplate
+    listKind: ProjectTemplateList
+    plural: projecttemplates
+    singular: projecttemplate
+  scope: Cluster
+  versions:
+    - name: v1alpha1
+      additionalPrinterColumns:
+        - jsonPath: .status.ready
+          name: Ready
+          type: boolean
+      schema:
+        openAPIV3Schema:
+          description: |
+            Describes the project template.
+
+            This resource is referenced by the [Project](cr.html#project) CR.
+          properties:
+            spec:
+              properties:
+                parametersSchema:
```

Comment:
Both `resourcesTemplate` and `parametersSchema` are required. Consider adding the 
```
required: ["parametersSchema", "resourcesTemplate"]
```

Reply:
Added